app/uk.gov.hmrc.helptosavestridefrontend/views/enter_customer_details.scala.html

```diff
@@ -22,184 +22,207 @@
-
-                <span class="form-hint">@messages("hts.customer-eligible.enter-details.form.personal-details.date-of-birth.hint")</span>
-                <div class="form-date">
-                    <div class="form-group form-group-day">
-                        <label for="dob-day" class="form-label  ">
-                            <span>@messages("hts.customer-eligible.enter-details.form.personal-details.date-of-birth.day.label")</span>
-                            <input type="number" class="form-control" name="dob-day" id="dob-day" value="">
-                        </label>
-                    </div>
-
-                    <div class="form-group form-group-month">
-                        <label for="dob-month" class="form-label">
-                            <span>  @messages("hts.customer-eligible.enter-details.form.personal-details.date-of-birth.month.label")</span>
-                            <input type="number" class="form-control" name="dob-month" id="dob-month" value="">
-                        </label>
-                    </div>
-
-                    <div class="form-group form-group-year">
-                        <label for="dob-year" class="form-label">
-                            <span>
-                            @messages("hts.customer-eligible.enter-details.form.personal-details.date-of-birth.year.label")
-                            </span>
-                            <input type="number" class="form-control" name="dob-year" id="dob-year" value="">
-                        </label>
+@defining(ApplicantDetailsForm.errorMessages(form)){ errorMessages: ApplicantDetailsForm.ErrorMessages =>
+  @defining(errorMessages.errors.toMap){ errorMessagesMap: Map[String,String] =>      
+        
+    @uk.gov.hmrc.helptosavestridefrontend.views.html.main_template(
+        title = messages("hts.customer-eligible.enter-details.title"),
+        bodyClasses = None,
+        isErrorPage = errorMessages.errorExists) {
+
+        @if(errorMessages.errorExists){
+            <div class="error-summary" role="alert" aria-labelledby="error-summary-give-nino" tabindex="-1">
+                <h2 class="heading-medium error-summary-heading" id="error-summary-give-nino">@messages("hts.customer-eligible.enter-details.error.title")</h2>
+                <ul class="error-summary-list">
+                @{errorMessages.errors.map{ case (fieldId, errorMessage) ⇒
+                    Html(s"""<li><a href= "#$fieldId">$errorMessage</a></li>""")
+                }}
+                </ul>
+            </div>
+        }
+
+
+        <h1>@messages("hts.customer-eligible.enter-details.title")</h1>
+        <p>@messages("hts.customer-eligible.enter-details.p1")</p>
+
+        @helpers.form(routes.StrideController.customerEligibleSubmit(), 'id -> "userDetailsForm") {
+
+            <h2>@messages("hts.customer-eligible.enter-details.form.personal-details.title")</h2>
+
+            @helpers.input_text(
```

Comment:
Inputs need the class 'form-control-error' when invalid so that they display the red border on the input: See https://govuk-elements.herokuapp.com/errors/